cmd/alleleSplit/toSam.go

```diff
@@ -9,12 +9,20 @@ import (
 	"strings"
 )
 
+func ASFilter(v vcf.Vcf, parentOne int16, parentTwo int16, F1 int16) bool {
```

Comment:
this is the same ASFilter() in cmd/filterGenotypes/filterGenotypes.go ?
If we are making identical functions is that when it's time to migrate it to a package? If not, when do we normally do that?

Reply:
Yeah, this seemed like a very specific function to me so it felt like it would be best if it was implemented where it was used.